### ISSUE TITLE
Vader Immortal || Not working:  crashing on start  || Windows mixed reality Samsung Odyssey 

### ISSUE THREAD

**Issue by ll-M-ll** [LibreVR/Revive#1662]
Hello,

Vader Immortal keeps crashing on start 

my system is  consists of :
 
 Vr headset : Windows mixed reality Samsung Odyssey .
 
Laptop:
 --------
 
 vga : Gtx 1050ti
 ram: 12 ddr4
 cpu: Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
 
 ------
 
 Setup
 ------------
 
 1) installed Oculus for the first time (on the default path )
 2) installed revive 2.1.0 (lastest) ( on the default path )
 3) downloaded and installed the game.
 
 
 Attempts:
 -----------
 >> attempt (1)
 >> .. before SteamVR update.
 
 1) started steamVR 
 2) injected the EXE file to Revive x64.
 
 >> Results:
 - the game launches and I can see the intro of Vader Immortal EP1  but then stops as ( press any key to continue )
 - the game didnt crash I had to alt+f4 to close it.
 
 - I should also mention that I got an error from SteamVR stating that action_manifest.jason was not found at ( <username>\Appdata\Local\Oculus\Input\action_manifest.json.
 
 >> attemp (2)
 same as attempt (1) just copied the files in revive to my game directory.
 
 >> same result 
 
 >> attempt (3)
 
 steamVR has been updated ( it didnt want to start without the update )
 
 Results >> game crashes on start without even showing anything but a black screen
 
 
 >> attempt (4)
 removed the game and installed it again also tried to us administrative privilege for both the game and revive.
 
 Result >> the same as attempt (3)
 
 does anyone know how to fix that or if there is any way around it ?
 
 thanks.

**Comment by ll-M-ll** [comment]
where is everybody ???

**Comment by ll-M-ll** [comment]
Update :

I've decided to try and reinstall it again.
I did that and i ended up in the same situation as attempt(1)

1- started steamVR
2- injected the EXE file to Revive x64.
Results:

the game launches and I can see the intro of Vader Immortal EP1 but then stops as ( press any key to continue )

it said on steamWindow ( couldnt find manifest bla bla bla at this path ) "...\Appdata\Local\Revive\Input\action_manifest.json"

on that note I closed the game ( alt+ f4 )

3 - copied the input folder from Revive's and past it on the mentioned path above 

4 - started the game again and I had a different issue this time 
" could not load binding .... bla bla bla "

so I'm always stuck at this screen where it says press anything to continue as nothing is being recognized.

![DarkImmortal](https://user-images.githubusercontent.com/43939767/104844474-18135600-58d9-11eb-88bf-8f53113c2605.jpg)
